packages/p2p-v2/src/components/BuySellForm/BuySellAmount/BuySellAmount.tsx

```diff
@@ -42,7 +42,7 @@ const BuySellAmount = ({
 
     return (
         <div className='flex flex-col gap-[2rem] py-[2.4rem]'>
-            <Text className='px-[2.4rem]' color='less-prominent' size={labelSize}>
+            <Text className='px-[2.4rem]' color='less-prominent' size='sm'>
```

Comment:
'sm' will be 10 in mobile view right? we need 12 for both..

Reply:
i checked in both the design and current prod, you need 14px for desktop, and 12px for responsive, and sm is 14px in desktop, 12px in responsive:

Desktop:
![Screenshot 2024-03-26 at 1 59 28 PM](https://github.com/binary-com/deriv-app/assets/103412909/585fb1ed-e1db-4520-99ae-3192b9848a46)


Responsive:
![Screenshot 2024-03-26 at 1 58 04 PM](https://github.com/binary-com/deriv-app/assets/103412909/8c3c4f8c-3924-4ff2-8c14-49dc380ba075)

Reply:
![Screenshot 2024-03-26 at 10 05 40 AM](https://github.com/binary-com/deriv-app/assets/122768621/baf13818-1ac2-4fd6-beb5-c6d1d9fb3d8b)
![Screenshot 2024-03-26 at 10 06 12 AM](https://github.com/binary-com/deriv-app/assets/122768621/428d9559-3597-402f-ae2e-511f50de3d48)

this is the one in production.. both are 12 px.. can you check again?

Reply:
Hmmm it really depends when we should be following prod, cause some of the designs are already updated and we should try to follow the behaviour from the designs or else qa will come after us 😅 

If there isnt design for any of the components, we can follow prod and inform PD team about it, but if there is, we should follow the design

Reply:
if we are going for the design, then there will be lot more changes needed again.. because for almost all the texts in the design, we have a different size for desktop and mobile. but for production, we have same size for the desktop and mobile and that's the reason we were going for isMobile ? 'something' : 'something else'

Reply:
true, but this is something we will face once they start doing QA tests, and we need to change as we work along the way before releasing. If there is any doubts, just raise it to the team, and if you feel strongly about certain designs should follow this way and such, then you can raise it as well 👀

The issue right now is that theres a lot of design inconsistencies between all the different figma files, and prod as well, but I think we can leave the current text as is since its following design